security_groups.tf

```diff
@@ -131,12 +131,12 @@ resource "exoscale_security_group_rules" "control_plane" {
   }
 }
 
-resource "exoscale_security_group" "worker" {
-  name        = "${var.cluster_id}_worker"
-  description = "${var.cluster_id} worker nodes"
+resource "exoscale_security_group" "infra" {
+  name        = "${var.cluster_id}_infra"
+  description = "${var.cluster_id} infra nodes"
 }
-resource "exoscale_security_group_rules" "worker" {
-  security_group = exoscale_security_group.worker.name
+resource "exoscale_security_group_rules" "infra" {
+  security_group = exoscale_security_group.infra.name
```

Comment:
not sure if I should just rename the existing security group + rules, or if there should be separate group for infra and worker each.

The reason for the rename is that I assume the ingress controller only need to run on infra nodes, therefore workers don't need to be part of this security group?

Reply:
There should be separate groups for regular workers and infra nodes, since the ingress router will only be hosted on the infra nodes in the default setup, and ports 80/443 don't need to be accessible from the LBs on regular workers.

Reply:
I re-added the security group for workers without any rules.

Reply:
I don't mind too much either way, but what line of thought caused you to re-add the worker security group without any rules? The general access rules are handled in security group "all_machines" anyway.

Reply:
I read
> There should be separate groups for regular workers and infra nodes

as "there should be two separate groups for each" i.e. one for workers, one for infra nodes, whereas only the infra nodes need the rules for the ingress. Looks like a misunderstanding though? :see_no_evil: